pkg/pxc/statefulset.go

```diff
@@ -54,7 +54,12 @@ func StatefulSet(sfs api.StatefulApp, podSpec *api.PodSpec, cr *api.PerconaXtraD
 	}
 
 	if cr.Spec.PMM != nil && cr.Spec.PMM.Enabled {
-		_, useAPI := secret.Data["pmmserverapi"]
+		useAPI := true
+		if _, ok := secret.Data["pmmserverkey"]; !ok {
+			if _, ok := secret.Data["pmmserver"]; ok {
+				useAPI = false
+			}
+		}
```

Comment:
could we move this logic to a method of `PMMSpec`?

Reply:
fixed https://github.com/percona/percona-xtradb-cluster-operator/pull/1119/commits/ec722f874a7389f98f366a46adeee384fccd9775

Reply:
I don't see the change. There are multiple places that uses same two if statements, if we make this a method we can use it everywhere